causalml/inference/tree/uplift.pyx

```diff
@@ -1366,7 +1366,14 @@ class UpliftRandomForestClassifier:
 
         '''
         # Make predictions with all trees and take the average
-        y_pred_ensemble = sum([tree.predict(X=X) for tree in self.uplift_forest]) / len(self.uplift_forest)
+
+        if self.n_jobs != 1:
+            y_pred_ensemble = sum(
+                Parallel(n_jobs=self.n_jobs, prefer="threads")
```

Comment:
Let's make the `joblib` backend as an optional input argument with `"threads"` as default to `__init__()` so that we can use e.g. `self.parallel_backend` in `fit()` and `predict()` instead of the hardcoded string.

Reply:
I completely agree. I propose to do this once #476 is merged. This way the new argument can be used both for `predict` and `fit`.

Reply:
Sounds good! Just merged #476. Thanks!